app/src/main/java/org/schabi/newpipe/database/playlist/PlaylistStreamEntry.kt

```diff
@@ -27,6 +27,7 @@ data class PlaylistStreamEntry(
         val item = StreamInfoItem(streamEntity.serviceId, streamEntity.url, streamEntity.title, streamEntity.streamType)
         item.duration = streamEntity.duration
         item.uploaderName = streamEntity.uploader
+        item.uploaderUrl = streamEntity.uploaderUrl ?: ""
```

Comment:
This should not have a default value of an empty String assigned, because of the same issue as in the database.

Reply:
How should it be done?

Reply:
Just simply remove the `?: ""`

Reply:
The `StreamInfoItem` is defined in the extractor and it isn't `Nullable`

Reply:
Fixed in https://github.com/TeamNewPipe/NewPipe/pull/6919/commits/6f1958d398f4a3a45d7f896bb03275fbaf497530